go.mod

```diff
@@ -6,35 +6,38 @@ require (
 	github.com/BurntSushi/toml v1.3.2
 	github.com/asaskevich/EventBus v0.0.0-20200907212545-49d423059eef
 	github.com/denisbrodbeck/machineid v1.0.1
-	github.com/doug-martin/goqu/v9 v9.18.0
+	github.com/doug-martin/goqu/v9 v9.19.0
```

Comment:
Let's make sure this file is excluded, since we are only touching documentation files